lib/model-interfaces/langchain/functions/request-handler/adapters/base/base.py

```diff
@@ -239,11 +259,11 @@ def run_with_chain_v2(self, user_prompt, workspace_id=None):
             # Used by Cloudwatch filters to generate a metric of token usage.
             logger.info(
                 "Usage Metric",
-                # Each unique value of model id will create a
-                # new cloudwatch metric (each one has a cost)
-                model=self.model_id,
-                metric_type="token_usage",
-                value=self.callback_handler.usage.get("total_tokens"),
+                extra={
```

Comment:
Changing the JSON format here would break the metric in the dashboard. Please undo
https://github.com/aws-samples/aws-genai-llm-chatbot/blob/main/lib/monitoring/index.ts#L289